package.json

```diff
@@ -3,20 +3,22 @@
   "version": "0.1.0",
   "description": "amplify-cli",
   "scripts": {
-    "test": "lerna run lint && lerna run test",
+    "test": "lerna run test",
+    "pretest": "lerna run lint ",
```

Comment:
Where do you add pretest tot he circle CI configs?

Reply:
You don't need to, `npm run test` will try to run `pretest`, `test` and `posttest` automatically